Add Connect Widgets & Dev Client App

SamEdwardes opened the pull request:
## What

This PR adds two new items of content:

- Connect Widgets app (See <https://colorado.rstudio.com/rsc/bike-predict-r-dashboard/> for preview of deployment)
- Dev Client app (See <https://colorado.rstudio.com/rsc/bike-predict-r-client-app-dev/> for preview of deployment)

## Related issues

- Closes #22

## Tasks

- [ ] After merging this PR redeploy the **Connect Widget** content from main branch on connect
- [ ] After merging this PR redeploy the **Dev App** content from main branch on connect

xuf12 reviewed (commented):
I think the thumbnails make different tabs look blended in with each other since they are the same colour. Other than that it looks good! 
<img width="955" alt="Screen Shot 2022-05-26 at 10 58 06 AM" src="https://user-images.githubusercontent.com/3258243/170573970-6b742d9b-6827-468e-ba87-2b77c601ad71.png">

SamEdwardes commented:
Hey Xu Fei - I updated the logos to be a little bit nicer. Still no amazing design... but I think it looks better:

<img width="723" alt="image" src="https://user-images.githubusercontent.com/18248949/170587226-37a2bec6-3f36-4fc2-b638-820ec33d2868.png">

I also removed the code from `content/01-etl/01-raw-data-refresh/document.qmd` and `content/02-model/01-train-and-deploy-model/document.qmd` that added a logo, and controlled permissions for the content that is deployed by connect into connect (e.g. pinned data, pinned model, and plumber API). Now, to be consistent all of the content is managed by the `_content_admin.qmd` script.

xuf12 reviewed (approved):
I think they look much better now. Graphic design is hard and I guess we can always improve them later on.
